internal/lldp/rx.go

```diff
@@ -30,21 +32,7 @@ var multicastAddrs = []string{
 	"01:00:0C:CC:CC:CC",
 }
 
-func (l *LLDP) setUpCapture() error {
-	l.mu.Lock()
```

Comment:
This method is now executed while already holding the lock inside `prepareCapture()` and it has setup all the Rx logger and rx TPacket before we get here.